components/kfp/Run_component/_samples/sample_pipeline.py

```diff
@@ -0,0 +1,20 @@
+from kfp import components
+
+kfp_endpoint = None
+
+
+run_component_or_pipeline_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/2d13b358690c761f64e5b59b70624de8f1f52a29/components/kfp/Run_component/component.yaml')
+
+
+def my_pipeline():
```

Comment:
Can we move the sample to samples top folder and add a test in the new sample test infra?

You can add a readme with link to the sample.

Reply:
I've created a feature request to add the sample to the tests: https://github.com/kubeflow/pipelines/issues/5755